### PR TITLE
2019 updates

### PR DESCRIPTION
**Pull request by JDurstberger**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -123,8 +123,3 @@ More information regarding accommodation can be found on the [Details page][Deta
 * [SoCraTes France][] / tbc, Rochegude, France
 * [SoCraTes BE][] / tbc, Tournai, Belgium
 
-
-<div style="margin-top:40px;">
-  <a href="/sponsorship.html" class="btn btn-primary btn-lg">Become a sponsor</a>
```
Comment (jukecraft) [inline index.md:158]: why do we want to remove this?

File: links.md [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+
+[Codurance]: http://www.codurance.com
+
+[SoCraTes Germany]: http://www.socrates-conference.de/
 [socrates_de_values]: https://www.socrates-conference.de/values
+[Farncombe Gallery]: http://farncombeconferencecentre.com/venue/gallery/
+[Conference Code of Conduct]: http://confcodeofconduct.com/
+[CodeFreeze]: http://www.codefreeze.fi/
+[SoCraTes Canaries]: https://socracan.com/
+[I T.A.K.E. Unconference]: http://itakeunconf.com/
```
Comment (jukecraft) [inline links.md:27]: I'm all for removing duplication and I love how this change cleared up some.

Moving these conference links somewhere else than where they're used might make it harder for someone to add/update one of them. Same goes for sponsorship links and maybe others that are only connected to a specific part of the page. Just a thought!

File: index.md (hunk 1 above)
Comment (JDurstberger) [reply index.md:158]: It was at the bottom of the page and I don't see how it added any value.
Sponsorship is constantly prominent on the side-bar and the main navigation.

Happy to put it back in if we think it adds value

File: links.md (hunk 2 above)
Comment (JDurstberger) [reply links.md:27]: I agree, that it means you need to look into 2 files.
I personally prefer to have a consistent way of working though, if we have most links in one file and some of them where they are used I fear people will start clutter them all over the place again

File: index.md (hunk 1 above)
Comment (jukecraft) [reply index.md:158]: I think it doesn't hurt :) We need more sponsors. But I'm not too fussed.

File: links.md (hunk 2 above)
Comment (jukecraft) [reply links.md:27]: yes, that's the tradeoff. I don't have a strong feeling about it.

**Pull request merged**